net/netfilter/nf_tables_api.c

```diff
@@ -8171,6 +8367,8 @@ static int nf_tables_commit(struct net *net, struct sk_buff *skb)
 	nft_commit_notify(net, NETLINK_CB(skb).portid);
 	nf_tables_gen_notify(net, skb, NFT_MSG_NEWGEN);
 	nf_tables_commit_audit_log(&adl, net->nft.base_seq);
+
+	nft_gc_seq_end(nft_net, gc_seq);
```

Comment:
This should be a part of ` netfilter: nf_tables: fix GC transaction races with netns and netlink event exit path`
https://github.com/ctrliq/kernel-src-tree/pull/2/commits/c3e22443802a82572d31ac50a84762bedf63928a

Reply:
Good catch - fixed in the next upcoming push.